runtime/src/main/java/org/corfudb/runtime/LogReplicationUtils.java

```diff
@@ -40,6 +40,29 @@ public final class LogReplicationUtils {
     // Retain the old name from LR v1 to avoid polluting the registry table with stale entries.
     public static final String REPLICATION_STATUS_TABLE_NAME = "LogReplicationStatus";
 
+    // /-------Constants specific to the RoutingQueue Model --------/
+
+    // Name of shared queue on the sender where log entry sync updates are placed transactionally
+    public static final String LOG_ENTRY_SYNC_QUEUE_NAME_SENDER = "LRQ_Send_LogEntries";
```

Comment:
is the constant format consistent with other fields. I can see the following 3 formats being used in this file

1) LogReplicationStatus
2) Log_Replication_Status
3) log_replication_status

Do we have some guidelines around the naming format?

Reply:
We do not have it publicized/written but names of stream tags use lower case and underscore.  Table names follow camel case.  @hisundar will also be able to share more details on naming conventions for Corfu entities.